docs/source/getting-started/path-variables.rst

```diff
@@ -68,28 +75,29 @@ make sure to change the ``java-version`` with the actual folder found on your co
 
         echo "export JAVA_HOME=/usr/lib/jvm/java-version" >> ~/.bash_profile
 
-In order for the changes to take effect you will need to restart your computer. Note that if you will be setting up the
-``PATH`` variable as well you can restart your computer after you're done with all the changes. Upon restarting check that
-the ``JAVA_HOME`` variable is set::
+Setting the PATH variable
+=========================
 
-        echo $JAVA_HOME
+Run the following command::
 
-The result should be the path to the JDK installation::
+        echo 'export PATH="$HOME/.daml/bin:$PATH"' >> ~/.bash_profile
 
-        /usr/lib/jvm/java-version
+Verifying the changes
+=====================
 
-Setting the PATH variable
-=========================
+In order for the changes to take effect you will need to restart the terminal,
+by closing the terminal program and starting it again.
 
-Run the following command::
+Please verify the JAVA_HOME variable by running::
 
-        echo 'export PATH="~/.daml/bin:$PATH"' >> ~/.bash_profile
+        echo $JAVA_HOME
```

Comment:
Hi @sofiafaro-da 

We (@moritzkiefer-da @robin-da) had a long discussion and have decided that because different terminals pick up the path settings differently telling the users to restart their computers is the easiest and safest thing to do.

Reply:
To expand on that: While you can certainly find ways around restarting, the kind of person that can do that reliably and knows what went wrong if they mess it up is the kind of person that won’t read those instructions anyway. For people less experienced with this, restarting might be annoying but at least it’s reliable and easy.

Reply:
I definitely disagree with this. Requiring the user to restart is a massive pain. (Recall that we have a goal of improving the user experience of the getting started guide...)

On Mac the instructions I gave are definitely sufficient (just restart the Terminal app). On Linux, I'm not sure, but I'd be much happier if we could avoid asking our users to restart their machines like we're installing a Windows XP app.

Reply:
> Recall that we have a goal of improving the user experience of the getting started guide...

I know but we changed this _because_ we’ve had a bunch of users that got confused by their shell environment and had issues with this. In the end, they solved it with a restart anyway.

I’m fine with changing it on MacOS if it works (but we should at least check that iterm2 does the same as terminal app) but I don’t think changing it on Linux is an improvement.

Reply:
It's a one-off thing that makes sure that PATH is set. It's not something that a user needs to do over and over so the "pain" of restarting is really minimal (if any). We had really a bunch of issues with making it work in different terminals. 

Now if we can _guarantee_ that this works 100% on MacOS it would be a great improvement